src/hotspot/share/gc/shared/partialArrayState.cpp

```diff
@@ -100,6 +100,8 @@ PartialArrayStateAllocator::Impl::~Impl() {
   for (uint i = 0; i < _num_workers; ++i) {
     _arenas[i].~Arena();
   }
+
+  FREE_C_HEAP_ARRAY(Arena*, _arenas);
```

Comment:
Bleh.  Thanks for catching this.  I don't see a need for the extra blank line, but otherwise this is good.